tests/unit/EntityInfo.spec.ts

```diff
@@ -93,14 +93,14 @@ describe('Entity Info component in a Correction as a named company', () => {
   })
 
   it('renders the business name and numbers', () => {
-    expect(wrapper.find('#entity-legal-name').text()).toContain('My Mock Name Inc.')
-    expect(wrapper.find('#entity-business-number').text()).toContain('1234567')
-    expect(wrapper.find('#entity-incorp-number').text()).toContain('BC1234567')
+    expect(wrapper.find('#entity-legal-name').text()).toBe('My Mock Name Inc.')
```

Comment:
This is more specific -- if something else in the markup adds text here, the unit test will tell us.